src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+- sources of on-chain randomness
+- concept of _safe head_ and _finalized blocks_
+
+For more information, check out this blog post by Tim Beiko on [How The Merge Impacts Ethereum’s Application Layer](https://blog.ethereum.org/2021/11/29/how-the-merge-impacts-app-layer/).
+</ExpandableCard>
+
+### Users and holders {#users-holders}
 
-Imagine Ethereum is a space ship that isn’t quite ready for an interstellar voyage. With the Beacon Chain the community has built a new engine and a hardened hull. When it’s time, the current ship will dock with this new system, merging into one ship, ready to put in some serious lightyears and take on the universe.
+<strong>You do not need to anything to protect your funds entering The Merge.</strong><br/><br/>
 
-## Merging with Mainnet {#docking-mainnet}
+<em>This bears repeating</em>: As a user or holder of ETH or any other digital asset on Ethereum, <strong>you do not need to do anything with your funds or wallet before The Merge.</strong>
 
-When ready, Ethereum Mainnet will "merge" with the Beacon Chain, becoming its own shard which uses proof-of-stake instead of [proof-of-work](/developers/docs/consensus-mechanisms/pow/).
+Despite swapping out proof-of-work, the entire history of Ethereum since genesis remains intact and unaltered after the transition to proof-of-stake. Any funds being protected by your own wallet before The Merge will still be accessible in the same manner after The Merge. No action is required to "upgrade" on your part.
 
-Mainnet will bring the ability to run smart contracts into the proof-of-stake system, plus the full history and current state of Ethereum, to ensure that the transition is smooth for all ETH holders and users.
+As we approach The Merge of Ethereum Mainnet, **you should be on high alert for scams trying to take advantage of users during this transition.** Do not send your ETH anywhere in attempt to "upgrade to ETH2." There is no "ETH2" token, and there is nothing more you need to do for your funds to remain safe.
+
+[More on Ethereum security](/security/#eth2-token-scam)
 
 ## After The Merge {#after-the-merge}
 
-This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. At this point Ethereum will be one step closer to achieving the full scale, security and sustainability outlined in its [Ethereum vision](/upgrades/vision/).
+This will signal the end of proof-of-work for Ethereum and start the era of a more sustainable, eco-friendly Ethereum. Learn more about [Ethereum energy consumption](/energy-consumption/).
+
+This will also set the stage for further scalability upgrades not possible under proof-of-work, bringing Ethereum one step closer to achieving the full scale, security and sustainability outlined in its [Ethereum vision](/upgrades/vision/).
+
+## Misconceptions about The Merge {#misconceptions}
+
+<ExpandableCard
+title="Misconception: &quot;Running a node requires staking 32 ETH.&quot;"
```

Comment:
Instead of repeating `Misconception:` a bunch of times, I might just rephrase these statements as questions, e.g.
- Does running a node require staking 32 ETH?
- Will The Merge will reduce gas fees?
- Can you withdraw staked ETH after The Merge?

Reply:
The goal here was to explicitly show these in terms of the things people are thinking, but are not correct. This approach was taken to try and catch peoples eyes, people who may think these things, but also make it absolutely clear these statements were false. @nloureiro If you have thoughts.

This is otherwise an FAQ again. We had spoke a bit about trying to make this more a "myth-busting" type section.
...Which does remind me, perhaps "myth" would be a shorter alternative here, at least in English.